### ISSUE TITLE
Steps 4-8 are Unclear

### ISSUE THREAD

**Issue by simonnuss** [MicrosoftDocs/powerbi-docs#1384]
There are several issues and I invite an author to complete refactor steps 4-8.  Examples include:

(1) Step 5:
    refers to "Power BI Premium service" - is it the "Power BI Premium" or "Power BI Service" object ID?  Be specific.  Moreover, capitalize the text to "Power Query Online" to maintain document consistency.
(2) Step 7:
    (i) has the user applying permissions to Power Query Online; step 8 also applies permissions to Power Query Online.  This is a critical typo.
    (ii) "For each of the two Power BI Premium Object IDs".  There is only one Premium Object ID. Rephrase this to "For each of the two Object IDs".
(3) Step 8 has a typo - it should read "step 5" and not "step 4"

See the original text below for reference:
"Step 5: Select and copy both Object IDs for Power BI Premium service and Power Query online from the results of your search. Be ready to paste those values in subsequent steps.

Step 7: For each of the two Power BI Premium Object IDs collected [above], Assign Read, Write, Execute Access and Default ACLs to your powerbi file system.

Step 8: For the Power Query Online Object ID collected in step 4, Assign Write, Execute Access and Default ACLs to your powerbi file system."

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: adddb13a-811b-4fda-dcf8-98ce19492126
* Version Independent ID: cc52ac03-b68e-f8c1-aecc-af0b09ccafc4
* Content: [Learn how to connect Azure Data Lake Storage Gen 2 to Power BI for dataflow storage - Power BI](https://docs.microsoft.com/en-us/power-bi/service-dataflows-connect-azure-data-lake-storage-gen2#feedback)
* Content Source: [powerbi-docs/service-dataflows-connect-azure-data-lake-storage-gen2.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/live/powerbi-docs/service-dataflows-connect-azure-data-lake-storage-gen2.md)
* Service: **powerbi**
* Sub-service: **powerbi-service**
* GitHub Login: @davidiseminger
* Microsoft Alias: **davidi**

**Comment by mihart** [comment]
Thanks for your comment, @simonnuss. This issue has been assigned to the doc author for review.

**Comment by davidiseminger** [comment]
Adding @bensack for further support on the process. We'll investigate and update the article as appropriate. Thanks!

**Comment by davidiseminger** [comment]
We've updated the document to help with clarifications and better steps. Hope that helps.
